packages/cozy-harvest-lib/src/helpers/doctypes.js

```diff
@@ -0,0 +1,5 @@
+export const TRIGGERS_DOCTYPE = 'io.cozy.triggers'
```

Comment:
Why not using directly the one from [here](https://github.com/cozy/cozy-libs/pull/275/files#diff-ef0cbadd46c6bfb4dae3a4859af864b6R4) instead of doing a dedicated file for that?